torch/nn/quantized/dynamic/modules/linear.py

```diff
@@ -62,7 +62,12 @@ def from_float(cls, mod):
         """
         assert type(mod) == NNLinear, 'nn.quantized.dynamic.Linear.from_float only works for nn.Linear'
         assert hasattr(mod, 'qconfig'), 'Input float module must have qconfig defined'
-        weight_observer = mod.qconfig.weight()
+        if mod.qconfig is not None and mod.qconfig.weight() is not None:
```

Comment:
Please add a test exercising from_float on a module with and without qconfig. (i.e a plain nn.module)

Reply:
Will add the unit test for this.

Reply:
Added the unit test.